app/Http/Requests/UserUpdateRequest.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Validation\Rule;
+
+class UserUpdateRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'name' => 'required|min:5|max:50',
+            'email' => [
+                'required',
+                'max:200',
+                Rule::unique('users')->ignore($this->id),
+            ],
+            'password' => 'required|min:8|max:50',
+            'avatar' => 'image|max:5120',
+        ];
+    }
+    
+    public function messages()
+    {
+        return [
+            'name.required' => 'Please enter name',
+            'name.min' => 'Name is too short (minimum is 5 characters)',
+            'name.max' => 'Name is too long (maximum is 50 characters)',
+            'email.required' => 'Please enter email',
+            'email.unique' => 'Email is unique',
```

Comment:
`This user email has already been used by another user`
ý a là dùng kiểu format thông báo này cho giống với những trường hợp unique trong hệ thống hiện tại

Reply:
a chưa thấy message này được định nghĩa trong spec. e bổ sung vào spec luôn nha